src/types/volunteer.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod';
+// import zEvents from './events'; <- import if made
+// import zHours from './hours'; <- import if made
+// import zDemographic from './demographic'; <- import if made
+
+const zVolunteer = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+  userName: z.string(),
+  id: z.number(),
```

Comment:
We don't need to include id as part to this model.

Reply:
I thought it was on the schema.
<img width="219" alt="Screenshot 2024-03-27 at 2 09 46 PM" src="https://github.com/hack4impact-utk/rooted-east/assets/117123520/64da0939-7bc3-4ca7-a28e-bb67347f56bd">

Reply:
removed